docs/hooks.md

```diff
@@ -0,0 +1,28 @@
+# Hooks
+
+## Extra scripts
+
+You can configure REMS to include extra script files by defining configuration
+
+```clj
+ :extra-scripts {:root "/tmp" :files ["/scripts/foo.js"]}
```

Comment:
The documentation should also say that the file needs to be located at /tmp/scripts/foo.js